src/comm_mpi2.F

```diff
@@ -457,19 +457,19 @@ subroutine exitt
          write(6,*) ' '
          write(6,'(A)') 'call exitt: dying ...'
          write(6,*) ' '
-         !call print_stack()
-         write(6,*) ' '
-         write(6,'(4(A,1p1e13.5,A,/))')
-     &       'total elapsed time             : ',ttotal, ' sec'
-     &      ,'total solver time incl. I/O    : ',ttime , ' sec'
-     &      ,'time/timestep                  : ',dtmp2 , ' sec'
-     &      ,'CPU seconds/timestep/gridpt    : ',dtmp1 , ' sec'
-#ifdef PAPI
-         write(6,'(2(A,1g13.5,/))')
-     &       'Gflops                         : ',dtmp3/1000.
-     &      ,'Gflops/s                       : ',papi_mflops/1000.
-#endif
-      endif
+c        call print_stack()
```

Comment:
Why not just remove this code? If we want it back then it's safely in the history.